core-web/libs/edit-content/src/lib/fields/dot-edit-content-category-field/utils/category-field.utils.ts

```diff
@@ -199,7 +199,7 @@ export const updateChecked = (
  * @param parentList
  */
 export const getParentPath = (parentList: DotCategoryParent[]): string => {
-    if (parentList) {
+    if (parentList.length > 0) {
```

Comment:
Using a guard statement is easier to read 

```javascript
export const getParentPath = (parentList: DotCategoryParent[]): string => {
    if (!parentList.length) {
        return '';
    }
    
    return parentList
        .slice(1)
        .map((parent) => parent.name)
        .join(' / ');
};
```

Reply:
Fixed!